main.go

```diff
@@ -144,3 +148,16 @@ func main() {
 		panic(err)
 	}
 }
+
+func loadInput(dumpFile string) *os.File {
+	if dumpFile != "" {
+		fmt.Printf("\nloading dump file from path: %s\n", dumpFile)
+		file, err := os.Open(dumpFile)
+		if err != nil {
+			fmt.Printf("\nerror reading file: %v err:%v", dumpFile, err)
+			panic(fmt.Errorf("cannot open file"))
```

Comment:
consider passing err itself to panic since it will capture more useful details like permission error, file not existent etc; not sure if we still need fmt.Printf in this case?

Reply:
Just followed the template of the other panic blocks.
In any case, changed here.